android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -473,21 +491,21 @@ constructor(
                 .flatMap { it.stratum }
                 .filter { it.hasValue() && it.value.hasText() }
                 .map { stratifier ->
```

Comment:
can we replace the 3 implicit loops above with 1 explicit loop and conditionals?